fda-model/stageDamage/ImpactAreaStageDamage.cs

```diff
@@ -37,8 +37,8 @@ public class ImpactAreaStageDamage : Validation, IReportMessage
         private ConvergenceCriteria convergenceCriteria;
 
         private int seed = 1234;
-        private int _numInterpolatedStagesToCompute = 10;
-        private int _numExtrapolatedStagesToCompute = 50;
+        private int _numInterpolatedStagesToCompute = 1;
+        private int _numExtrapolatedStagesToCompute = 5;
```

Comment:
I think we should bump these back up later

Reply:
totally fine with me.